Sources/Fuzzilli/FuzzIL/AbstractInterpreter.swift

```diff
@@ -219,6 +219,32 @@ public struct AbstractInterpreter {
         setType(of: v, to: t)
     }
 
+    private func calleeTypes(for signature: FunctionSignature) -> [Type] {
+
+        func processType(_ type: Type) -> Type {
+            if type == .anything {
+                return .unknown
+            } else if type == .iterable {
+                return environment.arrayType
+            } else {
+                return type
+            }
+        }
+
+        var types: [Type] = []
+        signature.parameters.forEach { param in
+            switch param {
+                case .plain(let t):
+                    types.append(processType(t))
+                case .opt(let t):
+                    types.append(processType(t) | .undefined)
+                case .rest(let t):
+                    types.append(processType(t) | .iterable)
```

Comment:
I think here we always just need `environment.arrayType` and no `processType(t)` at all. After all, a rest parameter will just be an array. Currently, we don't support nested array types (i.e. `.iterable(of: .integer)`) or so, but once we do, we'd need to update this logic. So maybe add a comment about that, too?